packages/scene-system/sdk/Fetch.ts

```diff
@@ -0,0 +1,19 @@
+import { PermissionItem, Permissions } from 'shared/apis/Permissions'
+
+export type FetchFunction = (resource: RequestInfo, init?: RequestInit | undefined) => Promise<Response>
+
+export function createFetch(permission: Permissions, originalFetch: FetchFunction) {
+  return (resource: RequestInfo, init?: RequestInit | undefined): Promise<Response> => {
+    const url = resource instanceof Request ? resource.url : resource
+    if (url.toLowerCase().substr(0, 8) !== 'https://') {
```

Comment:
This MAY be enabled tho in preview mode. With a warning I presume. It would be useful for development mode